cmd/influx/cli/cli.go

```diff
@@ -116,10 +116,14 @@ func (c *CommandLine) Run() {
 	if c.Execute != "" {
 		// Modify precision before executing query
 		c.SetPrecision(c.Precision)
-		if err := c.ExecuteQuery(c.Execute); err != nil {
-			c.Line.Close()
-			os.Exit(1)
+
+		// Make the non-interactive mode send everything through the CLI's parser
+		// the same way the interactive mode works
+		lines := strings.Split(c.Execute, "\n")
+		for _, line := range lines {
+			c.ParseCommand(line)
```

Comment:
We are ignoring the returned value which is a `bool` and indicates success or not.  I would suspect that we might want a `continue-on-error` flag or something that would tell the program that if you encounter an error, stop, or continue.

This could be added in a future PR.  It might be that `ParseCommand` should return an error now instead of a bool, as it's usage is evolving.